app/services/idv/profile_step.rb

```diff
@@ -28,6 +28,7 @@ def vendor_params
 
     def vendor_validate
       result = vendor_validator.validate
+      attempter.reset if attempter.window_expired?
```

Comment:
I believe the root cause of this is the incorrect logic in `Attempter#reset_attempts?`. The only thing that should determine whether or not to reset the attempts is the window expiration, right? It shouldn't matter how many attempts were made prior to the window expiration. If I get 3 chances every 24 hours, whether I've tried 1, 2,   or 3 times, if 24 hours have passed since my last attempt, I get to start at 0 again. 

Removing the line above, and doing this instead allows the tests to pass:
```ruby
def reset_attempts?
  window_expired?
end
```